spec/analytics_spec.coffee

```diff
@@ -1,23 +1,29 @@
 describe 'Analytics', ->
   before (done) ->
-    # mock ActionsManager
+    # mock PluginsManager, ActionsManager
+    requirejs.undef 'plugins_manager'
     requirejs.undef 'actions_manager'
+    @PluginsManager_spy = sinon.spy()
     @ActionsManager_spy = sinon.spy()
 
+    define 'plugins_manager', => @PluginsManager_spy
     define 'actions_manager', => @ActionsManager_spy
 
     require [
       'settings',
       'analytics',
       'session',
-      'actions_manager'], (Settings, Analytics, Session, ActionsManager) =>
+      'plugins_manager'
+      'actions_manager'], (Settings, Analytics, Session, PluginsManager, ActionsManager) =>
       @Settings = Settings
       @Analytics = Analytics
       @Session = Session
       @ActionsManager = ActionsManager
+      @PluginsManager = PluginsManager
```

Comment:
For readability try to keep the same sequence of dependencies in the assignments.